package.json

```diff
@@ -38,6 +38,9 @@
     "@digitalbazaar/data-integrity": "^2.5.0",
     "@digitalbazaar/data-integrity-context": "^2.0.1",
     "@digitalbazaar/ed25519-multikey": "^1.0.1",
+    "@digitalbazaar/did-method-key": "^5.2.0",
+    "@digitalbazaar/ed25519-signature-2020": "^5.4.0",
+    "@digitalbazaar/ezcap": "^4.1.0",
```

Comment:
Deps should be sorted.

Reply:
*after reading this seventeen times* 

OH! It's not that these three lines are incorrectly sorted/ordered in relation to each other (because these three *are* correctly ordered, in isolation), but that these three lines are incorrectly sorted/ordered *in relation to the lines above and below!*

*That's* several minutes I won't get back...